scripts/run_docker_build.sh

```diff
@@ -16,7 +16,7 @@ show_channel_urls: True
 CONDARC
 )
 
-cat << EOF | docker run -i \
+cat << EOF | docker run -i -t \
                         -v ${REPO_ROOT}/recipes:/conda-recipes \
                         -a stdin -a stdout -a stderr \
```

Comment:
This is where we are attaching stdout. What was the reason for this? Could we get rid of it and use a TTY instead?